src/test/java/com/synopsys/integration/blackduck/service/dataservice/UserServiceTestIT.java

```diff
@@ -30,123 +34,158 @@
-            userRequest.setLastName("IntegrationTest");
-            userRequest.setActive(true);
-            userRequest.setEmail("noreply@synopsys.com");
-            userRequest.setPassword("53CUR17y7hR0ugH085Cur17y");
+            projectView = PROJECT_SERVICE.getProjectByName(projectName).get();
 
-            HttpUrl userUrl = blackDuckApiClient.post(ApiDiscovery.USERS_LINK, userRequest);
-            userView = blackDuckApiClient.getResponse(userUrl, UserView.class);
+            userView = createUserView(userName, true);
 
-            List<UserView> projectUsers = projectUsersService.getUsersForProject(projectView);
+            List<UserView> projectUsers = PROJECT_USERS_SERVICE.getUsersForProject(projectView);
             assertFalse(projectUsers.contains(userView));
 
-            projectUsersService.addUserToProject(projectView, userName);
+            PROJECT_USERS_SERVICE.addUserToProject(projectView, userName);
 
-            projectUsers = projectUsersService.getUsersForProject(projectView);
+            projectUsers = PROJECT_USERS_SERVICE.getUsersForProject(projectView);
             assertTrue(projectUsers.contains(userView));
         } finally {
-            if (null != projectView) {
-                try {
-                    blackDuckApiClient.delete(projectView);
-                } catch (Exception ignored) {
-                    // ignored
-                }
-            }
-            if (null != userView) {
-                try {
-                    blackDuckApiClient.delete(userView);
-                } catch (Exception ignored) {
-                    // ignored
-                }
+            cleanView(projectView);
+            cleanView(userView);
+        }
+    }
+
+    @Test
+    public void testInvalidUser() throws IntegrationException {
+        String projectName = "user-group-project" + System.currentTimeMillis();
+        String userName = "user" + System.currentTimeMillis();
+        ProjectView projectView = null;
+
+        try {
+            ProjectSyncModel projectSyncModel = ProjectSyncModel.createWithDefaults(projectName, "test");
+            PROJECT_SERVICE.syncProjectAndVersion(projectSyncModel);
+
+            projectView = PROJECT_SERVICE.getProjectByName(projectName).orElse(null);
+
+            ProjectView finalProjectView = projectView;
+            assertThrows(BlackDuckIntegrationException.class, () -> PROJECT_USERS_SERVICE.addUserToProject(finalProjectView, userName));
+        } finally {
+            cleanView(projectView);
+        }
+    }
+
+    @Test
+    public void testGetActiveUsersForProject() throws IntegrationException {
+        String projectName = "user-group-project" + System.currentTimeMillis();
+        String inActiveUser = "inactive-user" + System.currentTimeMillis();
+        String activeUser = "active-user" + System.currentTimeMillis();
+        ProjectView projectView = null;
+        UserView inActiveUserView = null;
```

Comment:
I think 'inactive' is one word, so here, it seems like you are trying to declare a boolean flag as to whether or not you are in or out of 'active user view'